src/googleError.ts

```diff
@@ -94,6 +110,7 @@ export class GoogleError extends Error {
           statusDetailsObj.details.length > 0
         ) {
           error.statusDetails = statusDetailsObj.details;
+          delete error.details;
```

Comment:
This is a breaking change, is it really needed to delete this field?

Reply:
oh, thank you for pointing out. 
it does not need to be deleted fixing this issue. i'll remove that

Reply:
@alexander-fenster i removed that code. so only fallback logic exists in my PR